### DISCUSSION TITLE
[Lspsaga] how to make the UI better

### DISCUSSION THREAD

**Discussion by kayuxx** [nvimdev/lspsaga.nvim#1164] [Q&A]
i want to change the `lspsaga` **UI** from this:
![diag](https://github.com/nvimdev/lspsaga.nvim/assets/116302048/30552fbf-7425-42fa-8f19-ae802a8d542f)
to this:
![wn6lgpmj6ve91](https://github.com/nvimdev/lspsaga.nvim/assets/116302048/99d31202-0a4d-413a-957c-664652c6b565)
and for finder from this:
![finder](https://github.com/nvimdev/lspsaga.nvim/assets/116302048/4234ffd7-1375-4cb7-894a-9c7f531cff96)
to this:
![tss](https://github.com/nvimdev/lspsaga.nvim/assets/116302048/22d1102f-a494-4f2e-a7c9-2f33cf2ff28c)

i searched many times to get this **UI** and  I found nothing

**Comment by glepnir** [comment]
`hi SagaVirtLine ` to config it with a highlight. your `Comment` set italic. default it link to `Comment` .  before ui has many bug.

**Comment by kayuxx** [comment]
@glepnir i couldn't understand this one can you explain what is it `hi SagaVirtLine` and `Comment`?

**Comment by glepnir** [comment]
see your issue  #1165 and update.
